lib/bin/supllib/include/supl_session.h

```diff
@@ -0,0 +1,317 @@
+/*
+ * Copyright (c) 2019 - 2019, Nordic Semiconductor ASA
+ * All rights reserved.
+ *
+ * 1. By downloading this software, you acknowledge that all right, title and
```

Comment:
Is it really necessary to have this license here instead of the standard Nordic 5-clause? CC @bjorn-tore-taraldsen

Reply:
The library (.a) has to be distributed with this license and this is the .h file that get's delivered with this library (.a). It does not make sense to have different license on these two files.